server/models/ProfileModel.js

```diff
@@ -0,0 +1,69 @@
+const mongoose = require("mongoose");
+const bcrypt = require("bcryptjs");
+
+const profileSchema = new mongoose.Schema({
+  userId: {
+    type: String,
+    required: true,
+  },
+  username: {
```

Comment:
This field already exists in the `User` model, please remove it here

Reply:
I will remove